frameworks/Nim/nawak/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+export PATH="$IROOT/nim/bin:$IROOT/mongrel2/bin:$PATH"
+
+sed -i 's|host=.* port=5432|host='"${DBHOST}"' port=5432|g' model_postgre.nim
+
+echo "** Compiling app"
+nim c --threads:on -d:release -d:postgre_model -l:-Wl,-rpath,$IROOT/zeromq-4.0.3/lib -o:nawak_postgre app.nim
+
+echo "** Starting mongrel2"
+cd conf
+mkdir -p run logs tmp
+m2sh load -config mongrel2.conf
+m2sh start -name test &
```

Comment:
Travis is failing for nawak and I could find no error in its log file, but the tests timeout (which is consistent with either mongrel2 or nawak not being started/not finding each other).

As I can see nawak's startup log lines, I'm suspecting mongrel2 might not have started correctly?

The following are the [only lines that are output by mongrel2](https://s3.amazonaws.com/archive.travis-ci.org/jobs/45683237/log.txt), and it's from the `m2sh load -config mongrel2.conf` line.
```bash
INFO:run-ci:stop: Unknown instance: 
INFO:run-ci:Fri, 02 Jan 2015 21:29:55 GMT [WARN] (errno: None) No option --db given, using "config.sqlite" as the default.
```
Do you know why can't we see anything that `m2sh start -name test &` should output? There should be many lines starting with `$date GMT [INFO] ` like the following I usually get when starting mongrel2 without sudo on my local machine:
```bash
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/mime.c:49) MAX limits.mime_ext_len=128
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/host.c:74) MAX limits.url_path=256, limits.host_name=256
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/handler.c:343) MAX limits.handler_stack=102400
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/config/config.c:85) Loaded handler 1:tcp://127.0.0.1:9999:0138a43-micro-nimrod:tcp://127.0.0.1:9998:
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/config/config.c:227) Loaded route 1:/:handler for host 1:localhost
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/config/config.c:312) Loaded 1 hosts for server 1:bd81667c-505a-41da-952c-1e672338db8a
Fri, 02 Jan 2015 22:41:06 GMT [INFO] (src/server.c:445) LOADING Handler tcp://127.0.0.1:9999
[...]
```

All your changes look good by the way.